ipie/analysis/extraction.py

```diff
@@ -68,6 +68,49 @@ def extract_observable(filename, name="energy", block_idx=1):
         return walker_averaged.reshape((nsamp,) + tuple(obs_info["shape"]))
 
 
+def extract_hdf5_data_complex(filename, block_idx=1):
```

Comment:
Looks identical to extract_hdf5_data.  Is it any different from extract_hdf5_data? Why couldn't you re-use extract_hdf5_data?

Reply:
the other function writes only real values

Reply:
which function? I'm pretty sure we write estimators as complex values by default, so I'm still confused why we need both?

Reply:
https://github.com/JoonhoLee-Group/ipie/blob/develop/ipie/analysis/extraction.py#L33 ?

Reply:
Wait a second - why do we need to use hdf5 data at all? We are discontinuing the support for hdf5 output, aren't we?